tests/cephadm/test_service_stop.py

```diff
@@ -0,0 +1,25 @@
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify service stop operation for Mon and Mgr service
+    """
+    service_name = ["mgr", "mon"]
+
+    # Stop the service
+    node = ceph_cluster.get_nodes(role="installer")[0]
+    for service in service_name:
+        log.info(f"Stopping {service} service")
+        out = CephAdm(node).ceph.orch.stop(service)
+        log.info(f"Stopping {service} service operation executed")
+        expected_message = f"Stopping entire {service} service is prohibited."
```

Comment:
This can be modified as - 
```
        exp_out = f"Stopping entire {service} service is prohibited."
        if exp_out in out:
            log.info("Command executed with expected message")
            return 0
        
        log.info(f"Command failed with unexpected message '{out}'")
        return 1
```